package.json

```diff
@@ -48,34 +48,34 @@
     "cytoscape-spread": "^3.0.0",
     "cytoscape-svg": "0.4.0",
     "lodash": "^4.17.21",
-    "ramda": "^0.29.0",
-    "react": ">=16",
+    "ramda": "^0.29.1",
+    "react": ">=18",
     "react-cytoscapejs": "2.0.0",
-    "react-dom": ">=16",
-    "uuid": "^9.0.0"
+    "react-dom": ">=18",
+    "uuid": "^9.0.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.23.0",
-    "@babel/eslint-parser": "^7.22.15",
-    "@babel/preset-env": "^7.22.20",
-    "@babel/preset-react": "^7.22.15",
+    "@babel/core": "^7.23.7",
+    "@babel/eslint-parser": "^7.23.3",
+    "@babel/preset-env": "^7.23.8",
+    "@babel/preset-react": "^7.23.3",
     "babel-loader": "^9.1.3",
     "copyfiles": "^2.4.1",
-    "css-loader": "^6.8.1",
-    "eslint": "^8.50.0",
-    "eslint-config-prettier": "^9.0.0",
-    "eslint-plugin-import": "^2.28.1",
+    "css-loader": "^6.9.1",
+    "eslint": "^8.56.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-import": "^2.29.1",
     "eslint-plugin-react": "^7.33.2",
-    "npm": "^10.1.0",
-    "prettier": "^3.0.3",
-    "react-docgen": "^5.3.0",
-    "style-loader": "^3.3.3",
-    "webpack": "^5.88.2",
+    "npm": "^10.3.0",
+    "prettier": "^3.2.4",
+    "react-docgen": "^7.0.3",
```

Comment:
Ah, I had avoided updating `react-docgen` beyond 5.x, since it's used by the `extract-meta.js` script that lives only in the dash repo - but if component building still works with v7 then great! Still would be better for us to figure out how to make `extract-meta.js` use node modules from Dash itself... but that's not your problem 😅

Reply:
Thanks for pointing this out, @alexcjohnson ! I just realised that it's in fact not working with `"react-docgen": "^7.0.3"`... I'll revert this change

Reply:
Hmmm... Correct, we are not rebuilding the package during testing... I'll add it